packages/landing/src/config.map.ts

```diff
@@ -413,13 +413,22 @@ function addDefaultLayers(output: Map<string, LayerInfo>): void {
 
     {
       id: 'elevation',
-      title: 'Elevation',
+      title: 'Elevation DEM',
       projections: new Set([EpsgCode.Google]),
```

Comment:
I think we [support DEM in 2193 now already](https://github.com/linz/basemaps-config/pull/922). Might be good to add that in together.
```suggestion
      projections: new Set([EpsgCode.Nztm2000, EpsgCode.Google]),
```

Reply:
I don't think it's possible for this to be used by anything except Basemaps??